src/Service/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -122,11 +122,22 @@ public Type GetColumnSystemType(string columnName)
             }
             else
             {
-                throw new DataApiBuilderException(
+                if (MetadataProvider.IsDevelopmentMode())
+                {
+                    throw new DataApiBuilderException(
                     message: $"{columnName} is not a valid column of {DatabaseObject.Name}",
                     statusCode: HttpStatusCode.BadRequest,
                     subStatusCode: DataApiBuilderException.SubStatusCodes.BadRequest
                     );
+                }
+                else
```

Comment:
Can just evaluate message using conditionals since statuscode/substatuscode is same between the two exceptions.